hazelcast/src/test/java/com/hazelcast/nio/tcp/spinning/Spinning_TcpIpConnection_BasicTest.java

```diff
@@ -0,0 +1,20 @@
+package com.hazelcast.nio.tcp.spinning;
+
+import com.hazelcast.nio.tcp.TcpIpConnection_BasicTest;
+import com.hazelcast.nio.tcp.nonblocking.SelectNow_NonBlockingTcpIpConnectionThreadingModelFactory;
+import com.hazelcast.test.HazelcastSerialClassRunner;
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.Before;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+@RunWith(HazelcastSerialClassRunner.class)
+@Category(QuickTest.class)
```

Comment:
do we need to have this in the QuickTest category given it's an experimental feature?